pkgs/by-name/ta/tana/package.nix

```diff
@@ -0,0 +1,114 @@
+{ xorg
+, stdenv
+, lib
+, alsa-lib
+, at-spi2-atk
+, atkmm
+, cairo
+, cups
+, dbus
+, expat
+, glib
+, gtk3
+, libdrm
+, libglvnd
+, libxkbcommon
+, mesa
+, nspr
+, nss
+, pango
+, systemd
+, fetchurl
+, autoPatchelfHook
+, dpkg
+, makeWrapper
+, wrapGAppsHook
+}:
+let
+  xorgLibs = with xorg; [
+    libX11
+    libxcb
+    libXcomposite
+    libXdamage
+    libXext
+    libXfixes
+    libXrandr
+  ];
+  libs = [
+    alsa-lib
+    at-spi2-atk
+    atkmm
+    cairo
+    cups
+    dbus
+    expat
+    glib
+    gtk3
+    libdrm
+    libglvnd
+    libxkbcommon
+    mesa
+    nspr
+    nss
+    pango
+  ];
+  buildInputs = xorgLibs ++ libs;
+  version = "1.0.15";
+in
+stdenv.mkDerivation {
+  pname = "tana";
+  inherit version buildInputs;
+
+  src = fetchurl {
+    url = "https://github.com/tanainc/tana-desktop-releases/releases/download/v1.0.15/tana_${version}_amd64.deb";
+    hash = "sha256-94AyAwNFN5FCol97US1Pv8IN1+WMRA3St9kL2w+9FJU=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    dpkg
+    makeWrapper
+    stdenv.cc.cc.lib
+    wrapGAppsHook
```

Comment:
Is this double wrapping?

Reply:
My bad, I was testing something different at the beginning and forgot to remove those deps!